src/test/groovy/edu/ie3/datamodel/io/naming/EntityPersistenceNamingStrategyTest.groovy

```diff
@@ -124,7 +124,7 @@ class EntityPersistenceNamingStrategyTest extends Specification {
 		def fileName = "foo"
 
 		when:
-		fns.extractIndividualTimesSeriesMetaInformation(fileName)
+		fns.getIndividualTimesSeriesMetaInformation(fileName)
```

Comment:
Please consider renaming of `fns` to `ens` or something more meaningful (thoughout the whole test).